scripts/multi-docker-pull/README.md

```diff
@@ -0,0 +1,4 @@
+# multi-docker-pull
+Pull multiple Docker images at once and then tarball them
+
+This script will create a group of worker threads that will simultaneously pull docker images to your local docker instance.
```

Comment:
```suggestion
This script will create a group of worker threads that will simultaneously pull docker images to your local docker instance.

# requirements
- script tested with `docker==7.0.0` [pip package](https://pypi.org/project/docker/)
- script tested with `pigz 2.6`, [pigz](https://zlib.net/pigz/)
- script tested with Python 3.10
```

Reply:
That's a good point. I've been meaning to add a requirements.txt to this, but I keep forgetting. I'll do it later and commit this for now. Thanks for taking a look.